src/Compilers/CSharp/Portable/Symbols/ErrorTypeSymbol.ErrorTypeParameterSymbol.cs

```diff
@@ -161,15 +163,15 @@ public override int GetHashCode()
                 return Hash.Combine(_container.GetHashCode(), _ordinal);
             }
 
-            internal override bool Equals(TypeSymbol t2, TypeCompareKind comparison, IReadOnlyDictionary<TypeParameterSymbol, bool> isValueTypeOverrideOpt = null)
+            internal override bool Equals(TypeSymbol? t2, TypeCompareKind comparison, IReadOnlyDictionary<TypeParameterSymbol, bool>? isValueTypeOverrideOpt = null)
             {
                 if (ReferenceEquals(this, t2))
                 {
                     return true;
                 }
 
                 var other = t2 as ErrorTypeParameterSymbol;
-                return (object)other != null &&
+                return (object?)other != null &&
```

Comment:
>other [](start = 32, length = 5)

nit: consider using `is object`

Reply:
➡️ See https://github.com/dotnet/roslyn/pull/39192#discussion_r364483031